packages/sync/src/modules/WooCommerce.php

```diff
@@ -310,6 +311,21 @@ public function add_woocommerce_comment_meta_whitelist( $list ) {
 		return array_merge( $list, self::$wc_comment_meta_whitelist );
 	}
 
+	/**
+	 * Adds 'revew' to the list of comment types so Sync will listen for status changes on 'reviews'.
+	 *
+	 * @access public
+	 *
+	 * @param array $comment_types The list of comment types prior to this filter.
+	 * return array                The list of comment types with 'review' added.
+	 */
+	public function add_review_comment_types( $comment_types ) {
+		if ( is_array( $comment_types ) ) {
```

Comment:
Is this check necessary? I've noticed we don't do these on other filters that work in a similar way.